### PR TITLE
Maintenance warning

### PR DESCRIPTION
**Pull request by cgfarmer4**
Add maintenance warning.

Tests to Pass:
- Income + Credit test deprecation.
- BT fixes for eventlist.

No version update needed here. All config/test related.

### CODE REVIEW

File: src/test/java/com/plaid/client/integration/banktransfer/AbstractBankTransferTest.java [hunk 1]
```diff
@@ -64,6 +65,7 @@ public void setUpBankTransfer() throws Exception {
   protected abstract void bankTransferTest() throws AssertionError, Exception;
 
   @Test
+  @Ignore("Intermittently failing.")
```
Comment (aeidelson) [inline src/test/java/com/plaid/client/integration/banktransfer/AbstractBankTransferTest.java:68]: Should we get the BT team looking at these? Theoretically these shouldn't be failing, right?

**Review by aeidelson — APPROVED** [review]

File: src/test/java/com/plaid/client/integration/banktransfer/AbstractBankTransferTest.java (hunk 1 above)
Comment (cgfarmer4) [reply src/test/java/com/plaid/client/integration/banktransfer/AbstractBankTransferTest.java:68]: yea you're right, we should fix it. @bcambs09 any insight here?

Comment (bcambs09) [inline src/test/java/com/plaid/client/integration/banktransfer/AbstractBankTransferTest.java:68]:
```suggestion
```

Comment (bcambs09) [inline src/test/java/com/plaid/client/integration/banktransfer/AbstractBankTransferTest.java:68]:
```suggestion
```

File: src/test/java/com/plaid/client/integration/banktransfer/AbstractBankTransferTest.java [hunk 2]
```diff
@@ -12,6 +12,7 @@
 import java.util.List;
 import java.util.Arrays;
 import org.junit.Before;
+import org.junit.Ignore;
```
Comment (bcambs09) [inline src/test/java/com/plaid/client/integration/banktransfer/AbstractBankTransferTest.java:15]:
```suggestion
```

**Review by bcambs09 — COMMENTED** [review]
Not sure how easy it is to merge in this branch: https://github.com/plaid/plaid-java/pull/269, but that plus the suggested changes below should fix the issue

File: src/test/java/com/plaid/client/integration/banktransfer/AbstractBankTransferTest.java (hunk 1 above)
Comment (cgfarmer4) [reply src/test/java/com/plaid/client/integration/banktransfer/AbstractBankTransferTest.java]: Ty @bcambs09! ALl fixed.

**Review by davidzhanghp — APPROVED** [review]

**Pull request merged**